packages/cannon-worker-api/package.json

```diff
@@ -40,7 +40,7 @@
     "@types/three": "^0.139.0",
     "@typescript-eslint/eslint-plugin": "^5.17.0",
     "@typescript-eslint/parser": "^5.17.0",
-    "cannon-es": "^0.19.0",
+    "cannon-es": "^0.20.0",
```

Comment:
I noticed that `cannon-es` is listed as a `devDependency` but it reads to me like it should be a `dependency` so I'm wondering what I'm missing.

Reply:
`cannon-es` is bundled into the worker as part of our build process and does not need to be installed separately by the people using `use-cannon`.